src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostConfirm/ActualCostConfirm.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  Button,
+  Col,
+  KeyValue,
+  Modal,
+  ModalFooter,
+  Row,
+} from '@folio/stripes/components';
+
+import { getPatronName } from '../../util';
+
+import {
+  ACTUAL_COST_CONFIRM_MODAL_DEFAULT,
+  ACTUAL_COST_DEFAULT,
+  ACTUAL_COST_RECORD_FIELD_NAME,
+  ACTUAL_COST_RECORD_FIELD_PATH,
+  ACTUAL_COST_TYPES,
+  ACTUAL_COST_PROP_TYPES,
+  DEFAULT_VALUE,
+} from '../../../../../constants';
+
+import css from './ActualCostConfirm.css';
+
+const ActualCostConfirm = ({
+  setActualCostModal,
+  actualCostConfirmModal,
+  setActualCostConfirmModal,
+  actualCost,
+  setActualCost,
+}) => {
+  const {
+    isOpen,
+  } = actualCostConfirmModal;
+  const {
+    type,
+    actualCostRecord,
+    additionalInfo: {
+      actualCostToBill,
+      additionalInformationForStaff,
+      additionalInformationForPatron,
+    },
+  } = actualCost;
+  const patronName = getPatronName(actualCostRecord);
+  const instanceTitle = get(actualCostRecord, ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.INSTANCE_TITLE], DEFAULT_VALUE);
+  const materialType = get(actualCostRecord, ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.MATERIAL_TYPE], DEFAULT_VALUE);
+  const onKeepEditing = () => {
+    setActualCostConfirmModal(ACTUAL_COST_CONFIRM_MODAL_DEFAULT);
+    setActualCostModal({
+      isOpen: true,
+    });
+  };
+  const onConfirm = () => {
+    setActualCostConfirmModal(ACTUAL_COST_CONFIRM_MODAL_DEFAULT);
+    setActualCost(ACTUAL_COST_DEFAULT);
+  };
+  const renderFooter = (
+    <ModalFooter>
+      <Button
+        id="continueActualCost"
+        data-testid="continueActualCost"
+        buttonStyle="primary"
+        onClick={onConfirm}
+      >
+        <FormattedMessage id="ui-users.lostItems.modal.button.confirm" />
+      </Button>
+      <Button
+        id="cancelActualCost"
+        data-testid="cancelActualCost"
+        onClick={onKeepEditing}
+      >
+        <FormattedMessage id="ui-users.lostItems.modal.button.keepEditing" />
+      </Button>
+    </ModalFooter>
+  );
+  const getTitle = () => (
+    type === ACTUAL_COST_TYPES.BILL ?
+      <FormattedMessage
+        id="ui-users.lostItems.modal.bill.confirm.title"
+        values={{ patronName }}
+      /> :
+      <FormattedMessage
+        id="ui-users.lostItems.modal.doNotBill.confirm.title"
+        values={{ patronName }}
+      />
+  );
+
+  return (
+    <Modal
+      id="actualCost"
+      data-testid="actualCost"
+      label={getTitle()}
+      dismissible
+      open={isOpen}
+      onClose={onKeepEditing}
+      footer={renderFooter}
+    >
+      <Row>
+        <Col xs={12}>
+
+            <Row>
```

Comment:
@Dmitriy-Litvinenko would you mind fixing this lint issue?

Reply:
Fixed